test/unit/usb_connection.js

```diff
@@ -155,7 +155,7 @@ exports['USB.Connection.prototype._receiveMessages'] = {
 
     // The immediate return prevents these from being
     // called from in the error handler
-    test.equal(this.err.callCount, 0);
+    test.equal(this.warn.callCount, 0);
     test.equal(this.processExit.callCount, 0);
     test.done();
   },
```

Comment:
@rwaldron @johnnyman727 ... if they are necessary ... I think I didn't got the point yesterday ... for some reason I thought `logs.err` isn't legal ...